bin/zookeeper-server-stop.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
```

Comment:
Should we apply this change to ```zookeeper-shell.sh``` for consistency?

Reply:
@chia7712 Yes, I modified just now.